metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -2406,6 +2414,31 @@ private void updatePartitionInfo(
             newPartInfo.elr);
     }
 
+    private void validatePartitionReplicationFactorUnchanged(
+            PartitionRegistration part,
```

Comment:
Nit: Could you align the first argument with the opening bracket as other methods in this file?

Reply:
The code style in this class is somewhat inconsistent, and the formatting isn't fully aligned. That said, I can address this comment. Perhaps in the future, we could consider adding a checkstyle rule to standardize the argument alignment.